src/functions/delimsizing.js

```diff
@@ -275,7 +275,13 @@ defineFunction({
             middleDelim = delimiter.sizedDelim(
                 group.value.value, 1, options,
                 group.mode, []);
-            middleDelim.isMiddle = {value: group.value.value, options: options};
+
+            // Property `isMiddle` not defined on `span`. It is only used in
+            // this file above. Fixing this requires refactoring the htmlBuilder
+            // return type to support passing additional data (harder) or adding
+            // `isMiddle` to `span` just for this case (easier).
+            // $FlowFixMe
+            middleDelim.isMiddle = {value: group.value.value, options};
```

Comment:
Adding `isMiddle` to `span` doesn't seem like the right way to go b/c it's so specific to this particular situation.  The "harder" solution is probably the way to go.  Thanks for leaving a comment with suggestions o how to fix the `$FlowFixMe`.

Reply:
You're right. I updated it to identify which option is more ideal.